app/activity/[id]/component/activity-card.tsx

```diff
@@ -25,8 +24,24 @@ export const ActivityCard = ({ dummyActivityData }: ActivityCardProps) => {
           {dummyActivityData.title}
         </Heading>
         <Spacer />
-        <EditIcon />
-        <TrashIcon />
+
+        <Flex gap={{ base: '10px', md: '16px' }}>
+          <IconButton
+            aria-label="Edit this trip"
+            variant="roundIcon"
+            onClick={() => {}}
+            p={{ base: '6px', md: '10px' }}
+          >
+            <FiEdit3 size="100%" />
+          </IconButton>
+          <IconButton
+            aria-label="Delete this trip"
+            variant="roundIcon"
+            p={{ base: '6px', md: '10px' }}
+          >
+            <FiTrash2 size="100%" />
+          </IconButton>
+        </Flex>
```

Comment:
I changed the buttons for edit and delete as Slack
https://tabimemoworkspace.slack.com/archives/C063AKD40PL/p1702796450366749?thread_ts=1702776577.544669&cid=C063AKD40PL